keras_hub/src/models/yolo_v8/yolo_v8_detector_presets.py

```diff
@@ -0,0 +1,15 @@
+detector_presets = {
```

Comment:
Just have one list of presets. Only presets with weights (no weight free ones). We should be able to run this for all of them.

```python
keras_hub.Backbone.from_preset("some_preset")
keras_hub.Backbone.from_preset("some_preset", load_weights=False)
keras_hub.ImageObjectDetector.from_preset("some_preset")
```

This shouldn't require special setup here.